topics/kmm-plugin-releases.md

```diff
@@ -34,12 +34,39 @@ Release info
 Release highlights
 </th>
 <th>
-Compatible Kotlin plugin version
+Compatible Kotlin version
 </th>
 </tr>
 <tr>
 <td>
 
+**0.2.1**
+
+Released: February 15, 2021
+
+</td>
+<td>
+
+* [Support for Kotlin 1.4.30](https://youtrack.jetbrains.com/issue/KT-44038).
+* [New compatibility policy – for each major version of Kotlin there will be a new version of KMM plugin](https://youtrack.jetbrains.com/issue/KT-43211). (TODO TBD remove this item?)
+* [Ability to launch common tests for Android on a local JVM via run gutter](https://youtrack.jetbrains.com/issue/KT-42463).
+* [Provided class names for variables in the variable view in the Kotlin/Native debugger](https://youtrack.jetbrains.com/issue/KT-34630).
+* [Added test status to the gutter icon for multiplatform projects](https://youtrack.jetbrains.com/issue/KT-34504).
+* [Added the message that the KMM plugin works only on macOS for launching iOS-related things](https://youtrack.jetbrains.com/issue/KT-43442).
```

Comment:
things is not a proper word for technical documentation. 
`Notifying users that they can launch iOS applications only on macOS` 
I don't know whether we need to highlight this in release notes. It can be an exception but let's discuss on Monday

Reply:
I would prefer removing it as well.

Reply:
I almost agree, but think we need to consult with Anastasiia Kapanina just in case.